admin/class-convertkit-admin-settings-restrict-content.php

```diff
@@ -154,6 +154,18 @@ public function print_section_info() {
 		?>
 		<span class="convertkit-beta-label"><?php esc_html_e( 'Beta', 'convertkit' ); ?></span>
 		<p class="description"><?php esc_html_e( 'Defines the text and button labels to display when a Page, Post or Custom Post has its Member Content setting set to a Product, and the visitor has not authenticated/subscribed.', 'convertkit' ); ?></p>
+		<div class="notice notice-warning">
+			<p>
+				<?php
+				echo sprintf(
+					'%s %s %s',
+					esc_html__( 'If you have server side or a caching plugin enabled, you must configure it to disable caching when the', 'convertkit' ),
```

Comment:
Is this missing a word? Aren't all plugins server side?

Reply:
Reworded for clarity to `If your web host has caching configured (or you are using a caching plugin)`.  Caching might not be performed by a plugin (for example, [this ticket](https://convertkit.atlassian.net/browse/T3-276), where SiteGround have caching enabled in their hosting configuration).